changelogs/fragments/70449-facts-add-dst-timezone.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - facts - add new fact `date_time['tz_dst']`, which returns the daylight saving timezone (https://github.com/ansible/ansible/issues/69004).
```

The changelog fragments are complied into a restructured text document where double backticks are used to indicated incline code formatting.
```suggestion
  - facts - add new fact ``date_time['tz_dst']``, which returns the daylight saving timezone (https://github.com/ansible/ansible/issues/69004).
```